projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/ui-tests/tests/vdk-jupyterlab-extension.spec.ts

```diff
@@ -145,3 +145,26 @@ test('should create an init cell when opening a new notebook', async ({
     page.getByText(`job_input = VDK.get_initialized_job_input()`)
   ).toBeVisible();
 });
+
+test(
+  'should create a new dir, navigate to it,' +
+    'create a new job, attempt to create a job relative to' +
+    'the original dir and succeed',
+  async ({ page }) => {
```

Comment:
Shouldn't we be asserting something at the end of this test?

Reply:
no since the test would fail if any individual step is impossible

Reply:
> no since the test would fail if any individual step is impossible

Ok, but there's a possible scenario where all the commands pass and the job doesn't get created for some reason. If I deleted the call from the jupyter plugin to the code that actually creates the job, the test wouldn't catch it in the current state.